modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/UpNextQueue.kt

```diff
@@ -40,7 +41,7 @@ interface UpNextQueue {
         object Empty : State()
 
         // Loaded state includes the current episode as episode and the episodes in up next in queue. Queue does not include the currently playing episode
-        data class Loaded(val episode: BaseEpisode, val podcast: Podcast?, val queue: List<BaseEpisode>) : State()
+        data class Loaded(val episode: BaseEpisode, val podcast: Podcast?, val queue: List<BaseEpisode>, val useEpisodeArtwork: UserSetting<Boolean>) : State()
```

Comment:
This cannot be a part of the `UpNextQueue` state. They aren't related to each other.